helm/templates/job.yaml

```diff
@@ -45,7 +45,7 @@ spec:
       {{- toYaml . | nindent 8}}
     {{- end }}
       initContainers:
-        {{- if or (eq .Values.job.prefix "default") (eq .Values.job.prefix "attribute") }}
+        {{- if or (eq .Values.job.prefix "default") (eq .Values.job.prefix "attribute") (eq .Values.job.prefix "attribute-and-entity") }}
```

Comment:
Should we rather call it `all` instead of attribute-and-entity which sounds wierd